adapters/between/between_test.go

```diff
@@ -0,0 +1,12 @@
+package between
+
+import (
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"testing"
+)
+
+func TestJsonSamples(t *testing.T) {
+	// to do: change to a production test endpoint
+	//endpoint := "http://ads.betweendigital.com/s2s"
```

Comment:
Is this comment necessary? Would you consider removing it?

Reply:
removed, thanks